Move guidelines for builder tools and showcase into docs (#1017)
cardano-foundation/developer-portal: docs/portal-contribute.md

* [Issues in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/issues) 
* [Discussions in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/discussions)

### Add a project to showcase 
Help to add good projects to the [showcase section](http://developers.cardano.org/showcase/). Please note the guidelines for adding new projects:

> * It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but nothing for this showcase section.

Reviewer: ```suggestion
* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but not for this showcase section.
```
Have I understood this correctly? (i.e. a Forum not implemented in Cardano will _not_ be great for the Showcase section and therefore we will have _nothing_ of this kind here)

* It has to run on Cardano mainnet.
* It has to have a running product. (no presale, no protected pages, no coming soon messages)
* It has to have enough community reputation.

> * It has to provide a unique value from existing showcase items. 

Reviewer: ```suggestion
* It has to provide a unique value distinct from existing showcase items. 
```

* It has to have a stable domain name. (a random Netlify/Vercel domain is not allowed, no URL shortener, no app store links, or similar)
* The GitHub account that adds the project must not be new. 
* The GitHub account must have a history/or already be known in the Cardano community.
* Describe what makes your project special, avoid phrases like "the first this and that". Granular details like which project was first is tribal attribute known to cause rift and conflicts.

> * Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundrets of NFT. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TLDR: we are adding winners of the CNFT awards)

Reviewer: ```suggestion
* Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundreds of NFT projects. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TL;DR: we are adding winners of the [CNFT Awards](https://www.cnft-awards.io/))
```
TL/DR is considered canonical (although TLDR is also acceptable): https://www.merriam-webster.com/dictionary/TL%3BDR

> * If you add a project which main component is NFT, please select "nftproject" as tag. (not "nftsupport")

Reviewer: ```suggestion
* If you add a project with a main component of NFTs, please select "nftproject" as tag. (not "nftsupport")
```
